testing/test_junitxml.py

```diff
@@ -814,3 +814,44 @@ def test_pass():
         u'test_fancy_items_regression test_pass'
         u' test_fancy_items_regression.py',
     ]
+
+
+def test_testsuite_properties(testdir):
+    path = testdir.tmpdir.join("test_testsuite_properties.xml")
```

Comment:
I think so. Probably after the description of the `record_xml_property` fixture, before the *warning*... make sure to mention that it was introduced in 1.10 as well.